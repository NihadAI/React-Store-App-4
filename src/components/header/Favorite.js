import PropTypes from 'prop-types';

const Favorite = ({ favoriteCount }) => {
    return (
        <div className="header-top__favorite">
            <img className="header-top__favorite__img" src="/img/star-white.svg" alt="Favorite star" />
            <p className="header-top__favorite__count">({favoriteCount})</p>
        </div>
    );
};

Favorite.propTypes = {
    favoriteCount: PropTypes.number.isRequired,
};

export default Favorite;
