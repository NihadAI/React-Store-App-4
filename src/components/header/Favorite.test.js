import React from 'react';
import renderer from 'react-test-renderer';
import Favorite from './Favorite';

it('renders correctly', () => {
    const tree = renderer.create(<Favorite favoriteCount={5} />).toJSON();
    expect(tree).toMatchSnapshot();
});
