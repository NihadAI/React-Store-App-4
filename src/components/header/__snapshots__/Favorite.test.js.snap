// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header-top__favorite"
>
  <img
    alt="Favorite star"
    className="header-top__favorite__img"
    src="/img/star-white.svg"
  />
  <p
    className="header-top__favorite__count"
  >
    (
    5
    )
  </p>
</div>
`;
